python/res/job_queue/job_queue_node.py

```diff
@@ -39,7 +39,10 @@ def __init__(self,job_script, job_name, run_path, num_cpu,
         argc = 1
         argv = StringList()
         argv.append(run_path)
-        self.started = False
+
+        self.thread_status = ThreadStatusType.THREAD_READY
```

Comment:
This should be exposed as a read-only property.

Reply:
:+1: